scripts/us_bls/jolts/README.md

```diff
@@ -0,0 +1 @@
+# Scripts for importing dataset from the Search Results U.S. Bureau of Labor Statistics (BLS) Job Openings and Labor Turnover Survey (JOLTS)
```

Comment:
Can you add a bit more here? Can copy some important things from your import doc and include a command for running the scripts.